app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 skill_level: int) -> None:
+        super().__init__(nickname=nickname,
+                         favourite_dish=favourite_dish)
+        self._skill_level = skill_level
+
+    def player_info(self) -> str:
+        return f"Dwarf blacksmith" \
+               f" {self.nickname} with skill of the" \
+               f" {self._skill_level} level"

```

Comment:
```suggestion
        return (
            f"Dwarf blacksmith"
            f" {self.nickname} with skill of the"
            f" {self._skill_level} level"
        )
```
Use brackets instead of `\`, fix in all places